docs/development/plugins/data/public/kibana-plugin-plugins-data-public.esfilters.md

```diff
@@ -4,6 +4,13 @@
 
 ## esFilters variable
 
+> Warning: This API is now obsolete.
```

Comment:
Any reason not to deprecate this now?

Reply:
I think this would be considered a breaking change, no?

Reply:
Nah, deprecating something isn't a breaking change

Reply:
I changed the text to "removed"